### PR TITLE
Fix doxygen warnings.

### PR DESCRIPTION
**Pull request by dancerj**
Some parameters were undocumented.

tested with `doxygen doc/Doxyfile`

### CODE REVIEW

**Comment by dancerj** [comment]
Not sure if I want to remove the source code from individual pages or fix the 2 pages to include the source code.

File: include/fuse.h [hunk 1]
```diff
@@ -664,6 +667,10 @@ struct fuse_operations {
 	 */
 	int (*bmap) (const char *, size_t blocksize, uint64_t *idx);
 
+#if FUSE_USE_VERSION < 35
```
Comment (Nikratio) [inline include/fuse.h:670]: What's the rationale for this change? It doesn't seem related to Doxygen to me...

File: include/fuse.h [hunk 2]
```diff
@@ -984,6 +987,10 @@ int fuse_loop(struct fuse *f);
  */
 void fuse_exit(struct fuse *f);
 
+#if FUSE_USE_VERSION < 32
```
Comment (Nikratio) [inline include/fuse.h:990]: Same question here - why is this needed?

Comment (dancerj) [reply include/fuse.h:990]: the inline diff is a bit weird, but I've moved the comment line just before the function definition.

Before, the comment block was decorating 
int fuse_loop_mt_31(struct fuse *f, int clone_fd);

and there was a warning that `clone_fd` isn't described, and `config` isn't defined.

**Comment by dancerj** [comment]
I realized that I fixed the warnings but some descriptions disappeared. I figured out what I did wrong and here's a pull request to fix it: https://github.com/libfuse/libfuse/pull/608
